components/TimeSeries.client.vue

```diff
@@ -53,44 +54,42 @@ const { onMove } = useSynchronisableChart(
 );
 const { zIndex } = useAdjacentCharts(() => props.groupIndex, () => Number(appStore.timeSeriesGroups?.length));
 
-const seriesColors = colorBlindSafeLargePalette
-  .filter(c => ![plotBandsColorName, plotLinesColorName].includes(c.name))
-  .map(c => c.rgb);
-
 const chartContainerId = computed(() => `${props.timeSeriesMetadata.id}-container`);
 const yUnits = computed(() => timeSeriesYUnits(props.timeSeriesMetadata.id));
 const data = computed(() => getTimeSeriesDataPoints(appStore.currentScenario, props.timeSeriesMetadata.id));
 
 // https://mrc-ide.myjetbrains.com/youtrack/issue/JIDEA-118/
 const showCapacities = computed(() => props.timeSeriesMetadata.id === "hospitalised");
-const { initialCapacitiesPlotLines, minRange } = useCapacitiesPlotLines(showCapacities, appStore.capacitiesData, chart);
+const capacities = computed(() => appStore.currentScenario.result.data?.capacities.map((capacity) => {
+  const label = appStore.metadata?.results.capacities
+    .find(({ id: capacityId }) => capacityId === capacity.id)
+    ?.label || "";
```

Comment:
Could the label and id wrangling go into `useCapacitiesPlotLines`, to save both components doing it? Could pass the composable the result capacities, the color and the scenario id.

Reply:
Yes, done.

There were analogous refactors to do for intervention plot bands too, so I did those: moving id, color, label logic inside the intervention composable.